FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/DataFilter/R4ReferenceParser.cs

```diff
@@ -10,6 +10,7 @@
 using System.Text.RegularExpressions;
```

Comment:
Will not log error for diagnostic logger since ReferenceParseException here will be catch outside and will not throw.